### ISSUE TITLE
Rubocop 0.54.0 release

### ISSUE THREAD

**Issue by GeorgeKaraszi** [codeclimate/codeclimate-rubocop#119]
This one came exceptionally fast compared to prior versions.

**Comment by GeorgeKaraszi** [comment]
Any update on this?

**Comment by monfresh** [comment]
It would be greatly appreciated if someone from Code Climate could please provide an estimate or an update, even if it's just a quick "we are working on this." Is a channel addition something that can be automated so that it doesn't depend on someone whose full-time job isn't to maintain rubocop channels?

**Comment by toddmazierski** [comment]
Hello, all! My apologies for the delay. This version of RuboCop is now available on the new channel `rubocop-0-54`.

**Comment by Mange** [comment]
I can confirm that this is working for us.

**Comment by olbrich** [comment]
I just started getting this error..

```
Starting analysis
Running rubocop: Done!
Skipped rubocop: Channel rubocop-0-54 not found for rubocop, available channels: ["stable", "cache-support", "rubocop-0-42", "rubocop-0-46", "rubocop-0-48", "rubocop-0-49", "rubocop-0-50", "rubocop-0-51", "rubocop-0-52"]
```

with codeclimate 0.71.2

I could have sworn this worked a few days ago.

**Comment by toddmazierski** [comment]
Hi, @olbrich! I just released [v0.72.0](https://github.com/codeclimate/codeclimate/releases/tag/v0.72.0) of the CLI which contains the new `rubocop-0-54` channel. Mind upgrading and trying again?

**Comment by olbrich** [comment]
Seems to work, thanks!
